conf/application.conf

```diff
@@ -150,15 +150,8 @@ trader-test {
 }
 
 pagination {
-  arrival {
-    numberOfMovements = 20
-    numberOfErrorsPerPage = 20
-  }
-
-  departure {
-    numberOfMovements = 20
-    numberOfErrorsPerPage = 20
-  }
+  numberOfMovements = 20
```

Comment:
Don't forget these if we're changing config names :) We can probably remove them as we're never overriding the underlying value of 20

- https://github.com/hmrc/app-config-base/blob/9278a2c8d864d49a7cdcc06919056b4ad8325f2e/manage-transit-movements-frontend.conf#L109-L116
- https://github.com/hmrc/app-config-externaltest/blob/22cd332c3a572fd33cc377c3c49056aa9501bd87/manage-transit-movements-frontend.yaml#L50-L51
- https://github.com/hmrc/app-config-production/blob/313a3b0a3654e516e7315082bc9e90ba84f7544b/manage-transit-movements-frontend.yaml#L49-L50